VBA/Excel-VBA/articles/excel-performance-and-limit-improvements.md

```diff
@@ -1,247 +1,165 @@
+---
+title: Excel performance: Performance and limit improvements
+description: This article discusses performance improvements in Microsoft Excel 2016 and Microsoft Excel 2010. This article is one of three companion articles about techniques that you can use to improve performance in Excel as you design and create worksheets.
+ms.date: 9/28/2017
+---
 
-# Excel Performance: Performance and Limit Improvements
- **Summary:** This article discusses performance improvements in Microsoft Excel 2016 and Microsoft Excel 2010. This article is one of three companion articles about techniques that you can use to improve performance in Excel as you design and create worksheets.For more information about how to improve performance in Excel, see  [Excel Performance: Improving Calculation Performance](excel-improving-calcuation-performance.md) and [Excel Performance: Tips for Optimizing Performance Obstructions](excel-tips-for-optimizing-performance-obstructions.md).
+# Excel performance: Performance and limit improvements
 
-**Applies to:** Excel | Excel 2016 | Excel 2013 | Excel 2010 | Office 2016 | SharePoint Server 2010 | VBA
-
-**In this article**
-
-[Excel 2016 Performance Improvements](#xl2016PerfImp)
-
-[Excel 2010 Performance Improvements](#xl2010PerfImp)
-
-[Conclusion](#office2016excelperf_Conclusion)
-
-[About the Author](#xlAboutAuthors)
-
-[Additional Resources](#office2007excelperf_AdditionalResources)
-
-**Provided by:** ![MVP Contributor](images/mvp_avatar.jpg) Charles Williams,  [Decision Models Limited](http://www.decisionmodels.com/) | [About the Author](excel-improving-calcuation-performance.md#xlAboutAuthor)
-
-
-## Excel 2016 Performance Improvements
+## Excel 2016 performance improvements
```

Comment:
Let's remove this H2 as it's similar to the H1, and it's a stacked heading. Then we can elevate the H3s to H2s.